mutt_config.c

```diff
@@ -349,6 +349,9 @@ struct ConfigDef MainVars[] = {
   { "gecos_mask", DT_REGEX, &C_GecosMask, IP "^[^,]*", 0, NULL,
     "Regex for parsing GECOS field of /etc/passwd"
   },
+  { "greeting", DT_STRING, &C_Greeting, IP "", 0, NULL,
```

Comment:
`IP ""` -> `NULL` here too

Reply:
Compiler show warning here, when I changed `IP ""` to `NULL`:
```
mutt_config.c:352:41: warning: initialization of ‘long int’ from ‘void *’ makes integer from pointer without a cast [-Wint-conversion]
   { "greeting", DT_STRING, &C_Greeting, NULL, 0, NULL,
                                         ^~~~
```

Reply:
`0` is better, but the meaning is lost, as it seems to me